pkgs/applications/misc/keepassx/2.0-http.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, cmake, libgcrypt, qt5, zlib, libmicrohttpd, xorg, ... }:
+
+stdenv.mkDerivation rec {
+  name = "keepassx2-http-${version}";
+  version = "2.0.2-git"; 
```

Comment:
The package version parse is pretty strict ( https://nixos.org/nixpkgs/manual/#sec-package-naming ) and finishing with -git will break it.
It's also better to use unstable instead of git.
So, maybe something like:
```
name = "keepassx2-http-unstable-${version}";
version = "2.0.2"; 
```